Description: Correct pod NAME issues in LedgerSMB/Batch.pm
 Has a comma instead of a dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472542&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Batch.pm
+++ b/LedgerSMB/Batch.pm
@@ -1,5 +1,6 @@
 =head1 NAME
-LedgerSMB::Batch
+
+LedgerSMB::Batch - Batch/voucher management model for LedgerSMB 1.3
 
 =head1 SYNOPSIS
 Batch/voucher management model for LedgerSMB 1.3
